src/platform/Darwin/inet/InterfacesMonitor.mm

```diff
@@ -263,14 +263,31 @@ void GetInterfaceAddresses(nw_interface_t interface, chip::Inet::IPAddressType a
             });
         }
 
+        CHIP_ERROR InterfacesMonitor::StartMonitorPath(OnPathChange pathChangeBlock)
```

Comment:
Is it monitoring a single path, or multiple?  Seems like multiple?

Reply:
Obviously ...